### PR TITLE
Update rubocop_todo numbers.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -121,74 +121,73 @@ Style/LeadingCommentSpace:
...
 Style/MutableConstant:
   Enabled: false
 
-# Offense count: 51
+# Offense count: 54
 # Cop supports --auto-correct.
 Style/RedundantParentheses:
   Enabled: false
 
-# TODO: (Asher) fix these
```
Comment (joshlory) [inline .rubocop_todo.yml:152]: No longer a TODO? 😛

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -17,49 +17,49 @@ Lint/AmbiguousRegexpLiteral:
...
 # Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
 Lint/UnusedMethodArgument:
   Enabled: false
 
-# Offense count: 143
+# Offense count: 160
 # Configuration parameters: CountComments.
 Metrics/BlockLength:
-  Max: 605
+  Max: 309
```
Comment (joshlory) [inline .rubocop_todo.yml:37]: Looks like this was changed unintentionally, it trigged a Rubocop error after adding lines to routes.rb https://github.com/code-dot-org/code-dot-org/pull/13775/files#r105798755

File: .rubocop_todo.yml (hunk 1 above)
Comment (ashercodeorg) [reply .rubocop_todo.yml:152]: This whole page is a TODO(asher). ☹️

File: .rubocop_todo.yml (hunk 2 above)
Comment (ashercodeorg) [reply .rubocop_todo.yml:37]: Crap. Sorry, I missed that.

**Pull request merged**